tools/objc/libtool.sh

```diff
@@ -34,6 +34,17 @@ fi
 
 WRAPPER="${MY_LOCATION}/xcrunwrapper.sh"
 
+# Ensure 0 timestamping for hermetic results.
+export ZERO_AR_DATE=1
+
+if $(/usr/bin/dirname "$0")/libtool_check_unique "$@"; then
+  # If there are no duplicate .o basenames, libtool can be invoked with the original arguments
+  "${WRAPPER}" libtool "$@"
+  exit
+fi
+
+echo "Duplicate .o basenames detected, taking slow path"
```

Comment:
Probably want to remove this or this will always get logged for folks who are hitting this, and likely it will just be noise